src/entities/Pricebook.js

```diff
@@ -0,0 +1,24 @@
+const Entity = require('./Entity');
+class Pricebook extends Entity{
+    constructor(name, isStandardPricebook) {
+        super();
+        this._name = name;
+        this._isStandardPricebook = isStandardPricebook ?? false;
+    }
+
+    //Object getters
+    get standard() {
+        return this._isStandardPricebook;
+    }
+
+    get name() {
+        return this._name;
+    }
+
+    //Object setters
+    set name(newName) {
```

Comment:
Why not to just use `name` instead of `newName`?

Reply:
Good point. I thought it 'newName' would better describe the input but name could do just fine. Thank you for the feedback.